crates/data/src/config.rs

```diff
@@ -110,6 +110,16 @@ pub struct Config {
     #[serde(default = "reserved_cidrs")]
     exclude_cidr: Vec<IpNet>,
 
+    /// Enable listening via relay circuit through the gateway.
+    ///
+    /// When enabled (default), the scheduler establishes a listening address via the gateway's
+    /// relay circuit (/p2p-circuit). This allows workers to reach the scheduler even if it's
+    /// behind NAT or firewall.
+    ///
+    /// RECOMMENDATION: Keep enabled (true) unless the scheduler has public IP and external
+    /// addresses configured for direct connectivity.
+    relay_circuit: bool,
```

Comment:
Is there a special reason to make this configurable? Or is it just to aligne it with worker, data, ...